apps/web/pages/api/book/confirm.ts

```diff
@@ -115,6 +116,21 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       return res.status(400).json({ message: "booking already confirmed" });
     }
 
+    /** If we are confirming a booking that need pay and does not have payment,
+     * it would not have to save in google calendar */
```

Comment:
Google Calendar is just one of the Calendar integrations, so I think you could replace "google calendar" with "destination calendar" in the comment.